docs_source/usage/vocabs-unique-keys.md

```diff
@@ -0,0 +1,128 @@
+# A Vocabulary for Identifying Uniqueness of Array Items
+
+## 1. Purpose
+
+This document describes a vocabulary defining keywords that can be used to validate that arrary instances contain unique values as determined by one or more given paths within each item.
+
+The intent for these keywords is to cover the use cases discussed in the JSON Schema vocabularies GitHub issue [#22](https://github.com/json-schema-org/json-schema-vocabularies/issues/22).
+
+## 2. Declarations
+
+The ID for this vocabulary is `https://gregsdennis.github.io/json-everything/vocabs-unique-keys` (the URI to this document).
+
+A draft 2020-12 meta-schema which includes this vocabulary has been defined for convenience.  The ID for the meta-schema is `https://gregsdennis.github.io/json-everything/meta/unique-keys`, and it can also be found at this address.
+
+## 3. The `uniqueKeys` Keyword
+
+### 3.1 Syntax and Semantics
+
+The value of `uniqueKeys` must be an array of JSON Pointers as defined by [RFC 6901](https://tools.ietf.org/html/rfc6901).  At least one JSON Pointer MUST be specified.
+
+These JSON Pointers, when resolved against each item, define a set of values for that item.  If an item does not contain the location specified by a JSON Pointer, the implementation MUST keep a placeholder to indicate a missing value.
```

Comment:
By placeholder do you mean a null?

Reply:
Actually no.  I mean some internal representation that the value is missing rather than present with `null`.

Reply:
Okay. I guess we can't use null because that could be a legitimate value of a property being referenced.